Quartz/Engine/Include/Quartz/Core/Engine.hpp

```diff
@@ -42,22 +46,22 @@ namespace qz
 		{}
 	};
 
-	class Engine
+	enum class EngineOptions : uint
```

Comment:
This could be renamed to `EngineConfigFlags` or `EngineOptionFlags` to make it clear what they are and how they should be used.